### PR TITLE
New Recipe for Yoloxóchitl-Mixtec (SLR89)

### PR DESCRIPTION
**Pull request by ftshijt**
This PR is for the new recipe of Yoloxóchitl-Mixtec (100hrs endangered language).

- [x] Add RESULTS.md and upload model.

More suggestions are welcome.

### CODE REVIEW

**Comment by ftshijt** [comment]
I've got trouble understanding the circleCI error message, any suggestions?

**Comment by kamo-naoyuki** [comment]
It's format error.

```python
pip install black
black espnet
```

**Comment by ftshijt** [comment]
> ```python
> black espnet egs*/*/*/local
> ```

Thanks!

**Comment by ftshijt** [comment]
Ready to merge

File: egs/yoloxochitl_mixtec/asr1/run.sh [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+    data/${annotation_id}/remix_script.sh
+
+    # Kaldi Version Split
+    # ./utils/subset_data_dir_tr_cv.sh data/${annotation_id} data/${train_set} data/recog
+    # ./utils/subset_data_dir_tr_cv.sh --cv-spk-percent 50 data/recog data/${train_dev} data/${test_set}
+
+    # ESPNet Version (same as voxforge)
+    # consider duplicated sentences (does not consider speaker split)
+    # filter out the same sentences (also same text) of test&dev set from validated set
+    local/split_tr_dt_et.sh data/${annotation_id} data/${train_set} data/${train_dev} data/${test_set}
```
Comment (sw005320) [inline egs/yoloxochitl_mixtec/asr1/run.sh:86]: Is it OK?
If we don't have some official split, this way is fine.

Comment (ftshijt) [reply egs/yoloxochitl_mixtec/asr1/run.sh:86]: > LGTM!
> I have some minor comments.
> 
> * Do you think we should maintain Mixtec-consultant-database-unicode_2019-12-25.txt and speaker_wav_mapping_mixtec_remove_reserve.csv under the repository? One option is to include them in the openslr data. Either is fine, but I'm bit hesitating to maintain such data under the espnet repository.

Sounds good to add them into the openslr. I will contact dan to add the database code there and move them from the espnet repo

**Comment by ftshijt** [comment]
> LGTM!
> I have some minor comments.
> 
> * Do you think we should maintain Mixtec-consultant-database-unicode_2019-12-25.txt and speaker_wav_mapping_mixtec_remove_reserve.csv under the repository? One option is to include them in the openslr data. Either is fine, but I'm bit hesitating to maintain such data under the espnet repository.

Sounds good, I will contact to add the info into openslr and remove them from this pr.

File: egs/yoloxochitl_mixtec/asr1/run.sh (hunk 1 above)
Comment (ftshijt) [reply egs/yoloxochitl_mixtec/asr1/run.sh:86]: We currently do not have an official split. Do you think we should add one to the dataset? I think it's up to us

Comment (sw005320) [reply egs/yoloxochitl_mixtec/asr1/run.sh:86]: Maybe we can define it by ourselves by making a list of train/dev/eval splits?
The above script only considers the transcript overlap that happened in Voxforge and old commonvoice. But probably, we could have a good speaker-level and transcript-level open set.

**Comment by sw005320** [comment]
what is the status of this PR?

**Comment by ftshijt** [comment]
> what is the status of this PR?

I've contact openslr to add the files to the site. Once it is online, I will finalize this PR.

File: egs/yoloxochitl_mixtec/asr1/run.sh (hunk 1 above)
Comment (ftshijt) [reply egs/yoloxochitl_mixtec/asr1/run.sh:86]: Currently, there is 34 spk with 529 recordings (each corresponds to around 120 segmental utterances). In addition, I checked the overlapping rate for text. It seems that overlapping is not a problem if we divide them by recordings. The overlapping rate (the exact same utt number / total utt number) between recordings is around 0.5-1.5%.

I currently split them as:

set|spk | recordings
--|-- | --
train| 21 | 497
dev| 6 | 16
test |7 | 16

Any suggestions or comments?

Comment (sw005320) [reply egs/yoloxochitl_mixtec/asr1/run.sh:86]: I think this looks fine for me.
Let's try to standardize it.
Probably, we can make a webpage to briefly describe the database information, link for opneslr, and the official decision of the train/dev/test?

**Comment by sw005320** [comment]
Many thanks.
I'll just merge this PR.

**Pull request merged**